### PR TITLE
Generate .clr palette from JSON

### PR DESCRIPTION
**Pull request by aminalhazwani**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -27,4 +29,31 @@ function cssVars(content) {
...
 }
 
-gulp.task('colors', sequence('json2sass', 'json2css'));
+gulp.task('json2clr', function(cb) {
+  return gulp
+    .src('firefox-colors.json')
+    .pipe(change(toCapitalize))
+    .pipe(rename('firefox-colors-precompile.json'))
+    .pipe(gulp.dest('./'))
+    .pipe(exec('./json2clr -n "Firefox Colors" -i firefox-colors-precompile.json -o "Firefox Colors.clr"'));;
```
Comment (bwinton) [inline gulpfile.js]: I'm surprised there isn't a better way to run json2clr…

File: json2clr [hunk 2]
```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env xcrun swift
+// vim: ft=swift
+
+import Foundation
+import AppKit
```
Comment (bwinton) [inline json2clr]: This is not going to go so well on our CI infrastructure (or on my Windows box)…

File: gulpfile.js (hunk 1 above)
Comment (aminalhazwani) [inline gulpfile.js]: I tried to look for easier solutions but I couldn't find any.

File: json2clr (hunk 2 above)
Comment (aminalhazwani) [inline json2clr]: Yeah, good point. Now that I think of let me see if also windows has a global color palette format!

**Pull request merged**